platform/dolphin-platform-server/src/test/java/com/canoo/dolphin/server/context/DolphinContextTest.java

```
@@ -145,12 +146,17 @@ public void testNewDolphinCommands() {
     private final ClasspathScanner classpathScanner = new ClasspathScanner("com.canoo.dolphin");
 
     private DolphinContext createContext() {
-        return new DolphinContext(new DolphinPlatformConfiguration(), new DolphinSessionProvider() {
-            @Override
-            public DolphinSession getCurrentDolphinSession() {
-                return null;
-            }
-        }, new ContainerManagerMock(), new ControllerRepository(classpathScanner), new DefaultOpenDolphinFactory(), new DestroyCallbackMock(), new DestroyCallbackMock());
+        try{
+            return new DolphinContext(new DolphinPlatformConfiguration(), new DolphinSessionProvider() {
+                @Override
+                public DolphinSession getCurrentDolphinSession() {
+                    return null;
+                }
+            }, new ContainerManagerMock(), new ControllerRepository(classpathScanner), new DefaultOpenDolphinFactory(), new DestroyCallbackMock(), new DestroyCallbackMock());
+        }catch(ControllerValidationException cve){
+            return null;
```

Maybe we should throw the exception instead of returning null